packages/create-dapp/package.json

```diff
@@ -24,7 +24,7 @@
     "c8": "^9.1.0"
   },
   "dependencies": {
-    "agoric": "^0.21.1"
+    "agoric": "^0.22.0-u16.2"
```

Comment:
why does this use a different version than is in the repo? that breaks things like updating other dependencies

Reply:
you get an [error ](https://github.com/Agoric/agoric-sdk/issues/9852#issuecomment-2305472528) with `0.21.1` when you run `yarn create @agoric/dapp demo` using yarn4. I used this specific version as this was the latest published version on npm. 

> a different version than is in the repo? that breaks things like updating other dependencies

i'm confused. which repo?

Reply:
this repo, agoric-sdk. packages in a monorepo must work with the state of the other packages in the worktree. Exceptions are rare.

I think it's fair to say that `@agoric/dapp` does not support the PnP linker. Many packages don't. Yarn 4 using the nodeLinker shouldn't have any problem.

Reply:
> Yarn 4 using the nodeLinker shouldn't have any problem

Ideally, yes. But `yarn create` fails with an `ERR_MODULE_NOT_FOUND` if you're using yarn4 with agoric 0.21.1.

I will do some further digging to see if this can be fixed without bumping the package.

Reply:
If `yarn create` is really not compatible with HEAD of master, then that's a bug we need to fix. HEAD will always be released at some point